CHANGELOG.md

```diff
@@ -6,10 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.1.0]
+### Changed
+- [FEATURE] Manage transaction state ([#582](https://github.com/MetaMask/controllers/pull/582))
```

Comment:
Nit: The `[FEATURE]` prefix isn't needed here. The entry category ("Changed") should adequately describe what type of change this is.

Also, "Manage transaction state" doesn't do a great job of explaining the impact of this PR. Maybe something like.... "Improve transaction state management" would be better?